### PR TITLE
Search attribute merge edit

### PR DESCRIPTION
**Pull request by DJSanti**
Moved information on how to add and remove search attributes to the general term pages.

### CODE REVIEW

File: docs/tctl/admin/cluster/remove-search-attributes.md [hunk 1]
```diff
@@ -9,11 +9,25 @@ tags:
   - admin
 ---
 
-The `tctl admin cluster remove_search_attributes` command removes custom Search Attribute metadata from a given Cluster.
+> The Temporal tctl documentation covers version 1.16 of the Temporal CLI.
 
+The `tctl admin cluster remove-search-attributes` command removes custom Search Attribute metadata from a given Cluster.
```
Comment (djmagee) [inline docs/tctl/admin/cluster/remove-search-attributes.md:14]:
```suggestion
The `tctl admin cluster remove-search-attributes` command removes custom Search Attribute metadata from a Cluster.
```

**Comment by djmagee** [comment]
Looks like a link to `/tctl/how-to-add-a-custom-search-attribute-to-a-cluster-using-tctl` remains in `/tctl/index.md`. That might be what Netlify is complaining about.

File: docs/tctl/admin/cluster/remove-search-attributes.md [hunk 2]
```diff
@@ -9,11 +9,25 @@ tags:
...
 This operation has no effect on Elasticsearch index schema.
 
-Alias: `rsa`
+Use the following command to remove a <preview page={WhatIsASearchAttribute}>Search Attribute</preview> from a Cluster's metadata:
```
Comment (djmagee) [inline docs/tctl/admin/cluster/remove-search-attributes.md:17]: A leftover bit of MDX! Swap it for a garden-variety link.

File: docs/tctl/admin/cluster/add-search-attributes.md [hunk 3]
```diff
@@ -9,10 +9,25 @@ tags:
   - admin
 ---
 
-The `tctl admin cluster add_search_attributes` command allows custom search attributes to be added to a given Cluster.
-
 Alias: `asa`
 
+The `tctl admin cluster add-search-attributes` command allows Search Attributes to be added to a given Cluster.
```
Comment (djmagee) [inline docs/tctl/admin/cluster/add-search-attributes.md:14]:
```suggestion
The `tctl admin cluster add-search-attributes` command allows Search Attributes to be added to a Cluster.
```

**Review by flossypurse — APPROVED** [review]

**Pull request merged**